app/src/main/java/io/reark/rxgithubapp/data/stores/GitHubRepositorySearchStore.java

```diff
@@ -63,9 +63,23 @@ protected GitHubRepositorySearch read(Cursor cursor) {
 
     @NonNull
     @Override
+    protected ContentValues readRaw(Cursor cursor) {
```

Comment:
It's pretty much the same as for read(). I think it's safer to have it implemented explicitly, a default implementation that differs a bit from read() / getContentValuesForItem() could lead to subtle misbehaviour.